.pre-commit-config.yaml

```diff
@@ -1,91 +1,100 @@
-# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+# Copyright (c) 2022-2024, NVIDIA CORPORATION.
+ci:
+    autofix_commit_msg: "[pre-commit.ci] auto code formatting"
+    autofix_prs: false
+    autoupdate_branch: ""
+    autoupdate_commit_msg: "[pre-commit.ci] pre-commit autoupdate"
+    autoupdate_schedule: quarterly
+    skip: ["verify-alpha-spec"]
```

Comment:
I opened an issue for this: https://github.com/rapidsai/pre-commit-hooks/issues/58

@KyleFromNVIDIA If you have time and want to take a look, feel free. I don't think this is super important so I am okay with leaving the issue open and doing nothing in the short term.